src/components/ProductHeader.tsx

```diff
@@ -0,0 +1,18 @@
+interface Props {
+    productName: string;
+    price: string;
+    classOverrides?: Object;
+}
+
+const ProductHeader = (props : Props) => {
+    const {productName, price, classOverrides} = props;
+    const headerBSClasses = {margin: 'mx-1', ...classOverrides};
+    return (
+        <div className={Object.values(headerBSClasses).join(' ')}>
```

Comment:
You probably don't need object.values here. A spread operator would probably work, but I can't determine what `classOverrides` would look like since it's not being used, so I can't provide an example.